### PR TITLE
feat: one sdk log env

### PR DESCRIPTION
**Pull request by Jakub-Vacek**
## Description
Adds ONESDK_LOG= env variable to generated .env file with default value "off"
https://github.com/superfaceai/cli/issues/355

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have updated the documentation accordingly. For updating Oclif commands documentation use [oclif-dev](https://github.com/oclif/dev-cli#oclif-dev-readme).
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I haven't repeated the code. (DRY)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/logic/application-code/dotenv/dotenv.ts [hunk 1]
```diff
@@ -27,22 +28,25 @@ export function createNewDotenv({
...
   security?: SecurityScheme[];
   token?: string | null;
+  logEnabled?: boolean;
 }): NewDotenv {
   const previousContent = previousDotenv ?? '';
 
   const parameterEnvs = getParameterEnvs(providerName, parameters);
   const securityEnvs = getSecurityEnvs(providerName, security);
   const tokenEnv = makeTokenEnv(token);
+  const logEnv = makeLogEnv(logEnabled === true ? '"on"' : '"off"');
```
Comment (julpat) [inline src/logic/application-code/dotenv/dotenv.ts:45]: The motivation to change the default value is that user could have it already set up (e.g. using `export` in the terminal)?

File: src/logic/application-code/dotenv/dotenv.ts [hunk 2]
```diff
@@ -27,22 +28,25 @@ export function createNewDotenv({
...
   security,
   token,
+  logEnabled,
 }: {
   previousDotenv?: string;
   providerName: string;
   parameters?: IntegrationParameter[];
   security?: SecurityScheme[];
   token?: string | null;
+  logEnabled?: boolean;
```
Comment (julpat) [inline src/logic/application-code/dotenv/dotenv.ts:38]: You are not passing this new argument. So it's only for potential use in the future?

**Review by julpat — COMMENTED** [review]
Code review done.

So it's the "OR" variant, just to extended the `.env` template.
The option to promote this debugging flag should goes to OneSDK, right? The CLI `execute` command does not care.

File: src/logic/application-code/dotenv/onesdk-log.ts [hunk 3]
```diff
@@ -0,0 +1,2 @@
+export const ONESDK_LOG_ENV = 'ONESDK_LOG';
+export const ONESDK_LOG_COMMENT = 'Set to "on" to enable better output logging';
```
Comment (kysely) [inline src/logic/application-code/dotenv/onesdk-log.ts:2]: nitpick: I feel like “enable better logging” implies the default logging is not good. Based on [OneSDK's host docs](https://github.com/superfaceai/one-sdk/blob/247dceed182c93c9dd008b173879e009f7dd376b/host/javascript/README.md?plain=1#L149) I'd suggest the following description

```suggestion
export const ONESDK_LOG_COMMENT = 'Set to "on" to enable debug logging';
```

File: src/logic/application-code/dotenv/dotenv.ts (hunk 1 above)
Comment (kysely) [reply src/logic/application-code/dotenv/dotenv.ts:45]: I also don't see the point for this condition right now if the goal was to promote the env :)

**Review by kysely — COMMENTED** [review]
Looks good, left minor nitpick suggestions :)

**Review by kysely — APPROVED** [review]
LGTM 👍

**Pull request merged**